src/current/cockroachcloud/managing-cmek.md

```diff
@@ -326,10 +340,14 @@ To add a region to a cluster that already has CMEK enabled, update your cluster'
 
 ## Revoke CMEK for a cluster
 
-Revoking access to the CMEK means disabling all encryption/decryption of data in your cluster, which means preventing reading and writing any data from or to your cluster. This likely implies a shutdown of your service, with significant business implications. This is a disaster mitigation tactic to be used only in a scenario involving a severe, business critical security breach.
+When you revoke access to the CMEK, this disables all encryption and decryption of data at rest in your cluster and prevents access to the cluster's data until you restore access. In general, revoke access to the CMEK only when you are permanently decommissioning the cluster or to mitigate a critical security breach.
 
 Within your KMS platform, you can revoke access to the CMEK temporarily or permanently.
 
+{{site.data.alerts.callout_danger}}
```

Comment:
Does this callout make sense here, when it's specifically for Revoke CMEK?

Reply:
I truncated it to just the first sentence and just about revocation.

Reply:
But the point is just a few lines below we're actually recommending "revoke {{ site.data.products.dedicated }}'s access to your key at the IAM level in your cloud provider's KMS platform." - that's needed for the revoke to work. Which is why I wonder the need for this callout here.

Reply:
I can see what you mean. Removed.